h-include.js

```diff
@@ -41,35 +41,64 @@ var hinclude;
   hinclude = {
     classprefix: "include_",
 
-    show_content: function (element, req) {
-      var i, include, message, fragment = element.getAttribute('fragment') || 'body';
-      if (req.status === 200 || req.status === 304) {
-        var container = document.implementation.createHTMLDocument().documentElement;
-        container.innerHTML = req.responseText;
+    create_container: function (req) {
+      var container = document.implementation.createHTMLDocument().documentElement;
```

Comment:
Ok, I interpreted your previous comments that it would be some kind of negative side-effect of just adding an empty title (https://github.com/gustafnk/h-include/pull/18#discussion_r64164490). I'd be happy if you test this and submit a PR.